ED/protocol.html

```diff
@@ -666,30 +666,26 @@ <h2 property="schema:name">Hypertext Transfer Protocol</h2>
+                  <p><span about="" id="server-unauthenticated" rel="spec:requirement" resource="#server-unauthenticated"><span property="spec:statement">When a client does not provide valid credentials when requesting a resource that requires it (see <a href="#webid">WebID</a>), <span rel="spec:requirementSubject" resource="#Server">servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> send a response with a <code>401</code> status code (unless <code>404</code> is preferred for security reasons).</span></span></p>
 
-                  <p><span about="" id="server-authentication" rel="spec:requirement" resource="#server-authentication"><span property="spec:statement"><span rel="spec:requirementSubject" resource="#Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> conform to <cite>HTTP/1.1 Authentication</cite> [<cite><a class="bibref" href="#bib-rfc7235">RFC7235</a></cite>].</span></span> <span about="" id="server-unauthenticated" rel="spec:requirement" resource="#server-unauthenticated"><span property="spec:statement">When a client does not provide valid credentials when requesting a resource that requires it (see <a href="#webid">WebID</a>), <span rel="spec:requirementSubject" resource="#Server">servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> send a response with a <code>401</code> status code (unless <code>404</code> is preferred for security reasons).</span></span></p>
+                  <p><span about="" id="server-content-type" rel="spec:requirement" resource="#server-content-type"><span property="spec:statement"><span rel="spec:requirementSubject" resource="#Server">Server</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> generate a <code>Content-Type</code> header field in a message that contains content.</span></span></p>
```

Comment:
This requirement's fragment was `server-content-type-payload`. Can that persist?Alternatively, if we are going with "content", let's use that term for the sake of some specificity. Just as there is now `server-content-type-missing` - which is good.

```suggestion
                  <p><span about="" id="server-content-type-content" rel="spec:requirement" resource="#server-content-type"><span property="spec:statement"><span rel="spec:requirementSubject" resource="#Server">Server</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> generate a <code>Content-Type</code> header field in a message that contains content.</span></span></p>
```

Reply:
If more specificity is required, let us go with something more informative than repeating a word. For example: `#server-content-type-required`.

Reply:
No objection but what is not "required" per se? :) I'm fine with -content, -required, and perhaps preferably -included (which complement -missing)

Reply:
I meant it in the sense that that statement contains the extra MUST on top of RFC9110's SHOULD, so "required" rather than "recommended".

Reply:
That "required" may not give the desired hint. At least I didn't interpret that even while having the whole context in front of me. I think "includes" may be adequate (complements "missing") without needing additional information.

```suggestion
                  <p><span about="" id="server-content-type-includes" rel="spec:requirement" resource="#server-content-type"><span property="spec:statement"><span rel="spec:requirementSubject" resource="#Server">Server</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> generate a <code>Content-Type</code> header field in a message that contains content.</span></span></p>
```